resources/test/fixtures/flake8_simplify/SIM300.py

```diff
@@ -0,0 +1,9 @@
+# Bad
+
+'yoda' == compare # SIM300
```

Comment:
Couple other good test cases:

- `x == y and 'yoda' == compare` (should raise)
- `'yoda' == compare == 1` (shouldn't raise)